apps/ledger-live-desktop/src/renderer/reducers/application.ts

```diff
@@ -14,7 +14,7 @@ export type ApplicationState = {
   };
 };
 const { language, region } = getParsedSystemLocale();
-const osLangSupported = LanguageIds.includes(language);
+const osLangSupported = LanguageIdsNotFeatureFlagged.includes(language);
```

Comment:
`LanguageIds` no more used ?

[ASK] why do we need this `LanguageIdsNotFeatureFlagged` ?

Reply:
`LanguageIds` is still used as it contains all languages that Ledger Live can support.
`LanguageIdsNotFeatureFlagged` is necessary due to our current implementation. For example, when opening the app for the first time and setting the initial state of the app, we get the system language of the user and set it directly as the initial value. It is done in a function so we can't use the `useFeature` hook to check if a language is enabled or not. So for the initial state of the app we have to only use the languages that are not feature flagged otherwise it could cause some issues.